lib/improver/tests/utilities/test_cube_manipulation.py

```diff
@@ -704,14 +704,14 @@ def test_equalise_attributes(self):
                                     np.array([1000]))
         self.assertEqual(result[1].coord("model").points[0],
                          'Operational Mogreps UK Model Forecast')
-        self.assertNotIn("title", result[0].attributes.keys())
-        self.assertNotIn("title", result[1].attributes.keys())
-        self.assertAlmostEquals(result[0].attributes["grid_id"],
-                                result[1].attributes["grid_id"])
+        self.assertNotIn("title", list(result[0].attributes.keys()))
```

Comment:
Probably don't need these two `list`s
And the same thing several times below

Reply:
Done.